test/bdd/pkg/kms/kms_steps.go

```diff
@@ -172,3 +186,39 @@ func (s *Steps) checkSignMessageResp() error {
 
 	return nil
 }
+
+func (s *Steps) sendVerifySignatureReq(endpoint string) error {
+	postURL := strings.ReplaceAll(endpoint, "{keyEndpoint}", s.responseLocation)
+
+	req := fmt.Sprintf(verifySignatureReq, s.signature, s.message)
+	body := bytes.NewBuffer([]byte(req))
+
+	resp, err := bddutil.HTTPDo(http.MethodPost, postURL, contentType, body, s.bddContext.TLSConfig())
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	s.responseStatus = resp.StatusCode
+
+	respBody, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+	s.responseBody = respBody
+
+	return nil
+}
+
+func (s *Steps) checkVerifySignatureResp() error {
+	if s.responseStatus != http.StatusOK {
+		return fmt.Errorf("expected HTTP 200 OK, got: %d", s.responseStatus)
+	}
+
+	if len(s.responseBody) != 0 {
+		return fmt.Errorf("expected no error in the body, got: %q", string(s.responseBody))
```

Comment:
I'm curious as to why you used `%q` instead of `%s`?

Reply:
The response will have characters like `:`, so I want it to be quoted.

Reply:
What I meant was: why do you want them quoted?

Reply:
nothing special, just a matter of style :)

Reply:
@DRK3, is there any special reason why you would suggest to use `%s` instead of `%q`

Reply:
I think you shouldn't add unnecessary escaping... it will make the message harder to read if it gets output to the console

Reply:
@aholovko  `%s` should be the "default" choice. The `%q` format verb adds escape characters to the string. You don't want those escape characters there unless you need them for some reason...

Reply:
Avoid unnecessary escaping sounds like a valid reason... On the other hand, one can say exactly opposite - it helps to catch your attention to the actual response text and not to the auxiliary "expected no error in the body but ..." text. 🤔 

But let's just check.

With quoting:

<img width="858" alt="Screen Shot 2020-08-14 at 16 20 37" src="https://user-images.githubusercontent.com/239837/90253842-80519d80-de4a-11ea-8dc2-8e737a9e1e83.png">

Without:

<img width="841" alt="Screen Shot 2020-08-14 at 16 21 49" src="https://user-images.githubusercontent.com/239837/90253916-98c1b800-de4a-11ea-8dbb-4d81574df90b.png">

I am still not convinced that the latter is better. But if you say that you prefer the one without quotes I will remove them.

Reply:
I find again and again how wise was the choice made by the Go team regarding, for example, tabs...

@DRK3 Do you insist on switching to the unquoted version? I would like to complete with this PR and prepare the next one on Encrypt/Decrypt API.

Reply:
%q is adding those quote characters? That's interesting... I wasn't expecting that. The thing that I'm concerned about is what happens when you have a quote character in the text itself. That %q verb will add "/" characters, which is the part I'm trying to avoid

Reply:
What I would do is use %s and then surround the %s with quote characters. Then you can achieve both goals :)

Reply:
Talked about it with @aholovko, in this PR it doesn't matter but in the next one he'll add it since the response will have those characters that need escaping

Reply:
@DRK3 @aholovko 

> That %q verb will add "/" characters

If this is true then I am strongly against it because it'll make troubleshooting harder.